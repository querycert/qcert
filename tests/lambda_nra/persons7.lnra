/* Example of filter followed by map */

Persons.filter{ p => p.age = 32 }.map{ p => p.salary }.avg()

