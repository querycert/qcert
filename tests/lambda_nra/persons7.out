1166.6666666666667